target/linux/ath79/image/generic.mk

```diff
@@ -375,6 +375,7 @@ define Device/iodata_etg3-r
   ATH_SOC := ar9342
   DEVICE_TITLE := I-O DATA ETG3-R
   IMAGE_SIZE := 7680k
+  DEVICE_PACKAGES := -iwinfo -kmod-ath9k -wpad-basic
```

Comment:
iwinfo gets selected by [include/target.mk](https://github.com/openwrt/openwrt/blob/master/include/target.mk#L62) whenever a wpad-mini, wpad-basic, wpad or nas is selected. Since you are removing wpad-basic the iwinfo should not be there anymore.

Can you please test what happens if you remove "-iwinfo"? Does it work as it supposed to, or does the iwinfo package end up in the device's image? (Of course, you'll need to refresh your configuration when testing so
make sure to "remove the config" before each test and start out anew).
Thanks.

Reply:
Thank you for the review. I checked it in ```make menuconfig```, iwinfo is selected if ```-iwinfo``` is dropped from DEVICE_PACKAGES.
I'm compiling images with this change, I'll also check it later.

Reply:
The compiled image contains iwinfo if ```-iwinfo``` is dropped.

Reply:
> The compiled image contains iwinfo if -iwinfo is dropped.

Ok thanks, so the package is likely selected because the ath79 generic target.mk has ```DEFAULT_PACKAGES += wpad-basic```. This would probably also mean that you would need to add -libiwinfo since it will likely also get pulled. :thinking:

Reply:
Thank you, If ```DEFAULT_PACKAGES := -iwinfo -kmod-ath9k -wpad-basic```, libiwinfo is not selected.

```
$ echo "CONFIG_TARGET_ath79=y" > .config
$ echo "CONFIG_TARGET_ath79_generic=y" >> .config
$ echo "CONFIG_TARGET_ath79_generic_DEVICE_iodata_etg3-r=y" >> .config
$ make defconfig
Collecting package info: done
Collecting target info: done
(several dependency warnings)
#
# configuration written to .config
#
$ cat .config | grep "iwinfo"
# CONFIG_PACKAGE_libiwinfo-lua is not set
# CONFIG_PACKAGE_libiwinfo is not set
# CONFIG_PACKAGE_iwinfo is not set